### PR TITLE
Update page "Searching" regarding extended search unification

### PR DESCRIPTION
**Pull request by linuspahl**
This PR
- Adds an introduction for the search page
- Updates screenshots fo search time range selector
- Updated the saved search section
- Removes the sections histogram and analysis
- Copies widget, aggregations and message table fro "Extended Search" page to the "Search" page
- Updates the decorators section and its screenshots
- Updates the CSV export section and its screenshots
- Update the search result highlighting screenshot
- Removes dashboard section about statistical values, histogram, field values, action values, stacked charts and quick values from dashboard page

Next steps:
- Add info about widget creation / screenshots of the sidebar 
- Add info about widget -> copy to dashboard action
- Add info about aggregation interpolation
- Add info about aggregation event annotations
- Add info about search bar refresh actions
- Add info about sidebar fields + reference to value actions
- Rewrite / Update of dashboard page

Possible improvements
- Add better screenshot for search introduction page
- The order of the "Searching" subpages is pretty random, it would be great to have some kind of meaningful order. Also should we add a short explanation for subpages in the context of the overview (have a look at `/pages/queries.html`)

### CODE REVIEW

**Review by jalogisch — DISMISSED** [review]
👍 

Already like the structure and wording! We should check if splitting into sub-pages makes more sense to reduce the complexity.

**Comment by jalogisch** [comment]
Currently `extended search` and `searching` is having duplicated content - as this is now all in one we should merge all search related together and create sub pages. IMHO

File: pages/searching/configuration.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+================= ===========
+ISO 8601 duration Description
+================= ===========
+``P30D``          30 days
+``PT1H``          1 hour
+``P1DT12H``       1 day and 12 hours
+================= ===========
+
+More details about the format of ISO 8601 durations can be found `on Wikipedia <https://en.wikipedia.org/wiki/ISO_8601#Durations>`_.
```
Comment (cobaltclaudia) [inline pages/searching/configuration.rst:48]: Best to use reputable sites (i.e https://www.iso.org/iso-8601-date-and-time-format.html)

File: pages/searching/configuration.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+.. image:: /images/searching/queries_search_configuration.png
+
+All search configuration settings can be customized using the web interface on the *System* -> *Configurations* page in the *Search configuration* section.
+
+
+Query time range limit
+^^^^^^^^^^^^^^^^^^^^^^
+
+Sometimes the amount of data stored in Graylog is quite big and spans a wide time range (e. g. multiple years). In order to prevent normal users from accidentally running search queries which could use up lots of resources, it is possible to limit the time range that users are allowed to search in.
```
Comment (cobaltclaudia) [inline pages/searching/configuration.rst:14]: suggestion to swap `normal user` to `daily user` or `average user`

**Review by jalogisch — APPROVED** [review]
👍 

We have some wordings that might need to be polished - or I spot one wrong word. But I wish to push this documentation out. A wrong word is better than wrong documentation at all!

File: pages/searching/configuration.rst (hunk 2 above)
Comment (linuspahl) [reply pages/searching/configuration.rst:14]: I've updated the word to `daily user` here: https://github.com/Graylog2/documentation/pull/800

**Pull request merged**